Makefile

```diff
@@ -9,7 +9,7 @@ all: $(zipfiles)
 	@:
 
 %.zip: % %/node_modules
-	cd $< ; zip -x '*.tt' '*.yml' 'node_modules/.bin/*' -r $(abspath $@) .
+	cd $< ; zip -x '*.tt' '*.yml' 'node_modules/.bin/*' 'icon.png' -r $(abspath $@) .
```

Comment:
(this comment is about the actual icon file, but I can't comment on it)

The current icon used is our icon for small prints, like favicons etc. Please use our big one: https://github.com/home-assistant/home-assistant-assets/blob/master/logo-responsive.svg